Dockerfile

```diff
@@ -1,6 +1,7 @@
 # bump: alpine /FROM alpine:([\d.]+)/ docker:alpine|^3
 # bump: alpine link "Release notes" https://alpinelinux.org/posts/Alpine-$LATEST-released.html
-FROM alpine:3.20.0 AS builder
+ARG ALPINE_VERSION=alpine:3.20.0
+FROM $ALPINE_VERSION AS builder
```

Comment:
Also change `/FROM alpine:([\d.]+)/` above to something like `/ALPINE_VERSION=alpine:([\d.]+)/`.

If ok may i ask what is the reason you want to build with a different version? there has been some build workaround for older version of alpine that have been cleaned up once they were not needed. But i guess you will notice if things break. The only other things i can think of is security issues and maybe that older versions of musl have had some dns problems.

Reply:
Aha I see. The extra certificatees are needed during build? otherwise i guess you could do something like:

```Dockerfile
FROM alpine:3.20.0
RUN apk add ca-certificates --no-cache --no-check-certificate
COPY my-certificate.crt /usr/local/share/ca-certificates
RUN update-ca-certificates
COPY --from=custom-ffmpeg /ffmpeg /usr/local/bin
```

```sh
# build ffmpeg with Fraunhofer FDK AAC codec 
docker build --build-arg ENABLE_FDKAAC=1 --build-arg -t custom-ffmpeg

# build alpine base image
docker build -t custom-alpine .

Reply:
I need the certificate during build time, because my organization uses TLS inspection and I have to add (or trust) their certificate just to download something.

Reply:
Then it makes sense, thanks for explaining